hazelcast/src/test/java/com/hazelcast/query/impl/getters/GetterFactoryTest.java

```diff
@@ -58,6 +77,15 @@ public void newFieldGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldT
     }
 
     @Test
+    public void newMethodGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldType() throws Exception {
```

Comment:
what I normally do when running into very long test method-names, it to create a subclass of the test that only deals with e.g. method getter. So this part doesn't need to be repeated in the name.